File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/fingo_error/FingoErrorResponse.java
package com.yelloco.fingodriver.models.networking.fingo_error;

import java.util.ArrayList;
import java.util.List;

public class FingoErrorResponse
{
    private List<FingoErrorObject> fingoErrorList;

    public FingoErrorResponse(){
        this.fingoErrorList = new ArrayList<>();
    }

    public FingoErrorResponse(List<FingoErrorObject> fingoErrorList) {
        this.fingoErrorList = fingoErrorList;
    }

    public List<FingoErrorObject> getFingoErrorList() {
        return fingoErrorList;
    }

    public void setFingoErrorList(List<FingoErrorObject> fingoErrorList) {
        this.fingoErrorList = fingoErrorList;
    }
}

File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/fingo_error/FingoErrorResponse.kt
package com.yelloco.fingodriver.models.networking.fingo_error

import java.util.*

class FingoErrorResponse
{
    var fingoErrorList: List<FingoErrorObject>

    constructor() {
        fingoErrorList = ArrayList()
    }

    constructor(fingoErrorList: List<FingoErrorObject>) {
        this.fingoErrorList = fingoErrorList
    }

    override fun toString(): String {
        return "FingoErrorResponse(fingoErrorList=$fingoErrorList)"
    }


}